aws/alarms/cloudwatch.tf

```diff
@@ -61,15 +61,19 @@ resource "aws_cloudwatch_metric_alarm" "ELB_5xx_error_warn" {
 resource "aws_cloudwatch_metric_alarm" "UnHealthyHostCount" {
   alarm_name          = "UnHealthyHostCount-SEV1" # SEV1 will prompt the on-call team to respond.
   alarm_description   = "ELB Health Check - UnHealthyHostCount exceed threshold."
-  comparison_operator = "GreaterThanThreshold"
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  threshold           = "1" # If there is at least one unhealthy host
   evaluation_periods  = "1" # Evaluate once
-  metric_name         = "HTTPCode_ELB_5XX_Count"
+  metric_name         = "UnHealthyHostCount"
   namespace           = "AWS/ApplicationELB"
   period              = "60"           # Every minute
   statistic           = "SampleCount"  # use the number of data points during the period
-  threshold           = "1"            # If there is at least one unhealthy host
   treat_missing_data  = "notBreaching" # don't alarm if there's no data
   alarm_actions       = [var.sns_topic_alert_critical_arn]
+  dimensions = {
+    LoadBalancer = var.lb_arn_suffix
+    TargetGroup  = var.lb_target_group_1_arn
```

Comment:
I'm not sure if we should also monitor Target Group 2, which is part of the blue/green deployment

Reply:
Good catch - you will need an alarm for both target groups to catch unhealthy hosts no matter which is active.